### PR TITLE
Translate strings to Russian for iOS widgets and watch

### PR DESCRIPTION
**Pull request by De1ain**
Related issue  #88

### CODE REVIEW

**Comment by De1ain** [comment]
@guytepper, I think that's it.
Please have a look.

File: ios/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
 
-"NEXT TRAIN" = "הרכבת הבאה";
-"UPCOMING" = "הרכבות הבאות";
-"TOMORROW" = "מחר";
-"ARRIVAL" = "הגעה";
-"PLATFORM" = "רציף";
-"TRAIN NO." = "רכבת מס׳";
-"TRAIN" = "רכבת";
+"NEXT TRAIN" = "Следующий поезд";
+"UPCOMING" = "Следующие поезда";
```
Comment (guytepper) [inline ios/ru.lproj/Localizable.strings:17]: Is there a shorter synymom for those? Currently they wrap to 2 lines:
Screen Shot 2022-03-10 at 10 29 00

Screen Shot 2022-03-10 at 10 30 26

File: ios/ru.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+"TOMORROW" = "Завтра";
+"ARRIVAL" = "Прибытие";
+"PLATFORM" = "Платформа";
+"TRAIN NO." = "Поезд номер";
```
Comment (guytepper) [inline ios/ru.lproj/Localizable.strings:21]: Is there a shorter string for this?
Screen Shot 2022-03-10 at 10 29 00

**Review by guytepper — COMMENTED** [review]
Thank you 🙏 It looks great. 
See my comments, if you could find shorter strings for these it'd be great!

**Comment by guytepper** [comment]
Looks great now :) 
btw, we do need to remove those lines - they actually reference a deleted file

Screen Shot 2022-03-10 at 13 39 56

**Pull request merged**